projects/angular/src/utils/i18n/common-strings.interface.ts

```diff
@@ -301,4 +301,8 @@ export interface ClrCommonStrings {
   fileCount: string;
   clearFile: string;
   clearFiles: string;
+
+  // Tree node
```

Comment:
Same as the other.

Reply:
Updated in https://github.com/vmware-clarity/ng-clarity/pull/1575/commits/a89de534c1ce7d9bfcc03e67522c1ee5a64de206